shiksha-website/shiksha-frontend/src/app/shared/components/baseline-survey/baseline-survey.component.scss

```
@@ -1,618 +1,569 @@
-}
-
-/* SELECT STYLING */
-.select-wrapper {
-  position: relative;
-  width: 100%;
-}
+// @import url('https://fonts.googleapis.com/css2?family=Baloo+2:wght@500;600;700&family=Inter:wght@400;500;600;700&display=swap');
+
+/* Make the Angular Material dialog hug the wizard card */
+.survey-wrapper {
+  :host ::ng-deep .mat-dialog-container {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)